edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -105,6 +105,8 @@ public static class Constants
 
         public const string WorkloadApiVersion = "2020-10-10";
 
+        public const string HttpClientTimeoutSecs = "httpClientTimeoutSecs";
```

Comment:
We should call it something like WorkloadApiTimeoutSecs

Reply:
I agree, (though it is managementAPI)